auth/api/iam/api.go

```diff
@@ -170,18 +170,21 @@ func (r Wrapper) IntrospectAccessToken(ctx context.Context, request IntrospectAc
 	// Validate token
 	if request.Body.Token == "" {
 		// Return 200 + 'Active = false' when token is invalid or malformed
+		log.Logger().Debug("IntrospectAccessToken: missing token")
 		return IntrospectAccessToken200JSONResponse{}, nil
 	}
 
 	token := AccessToken{}
 	if err := r.accessTokenStore().Get(request.Body.Token, &token); err != nil {
 		// Return 200 + 'Active = false' when token is invalid or malformed
+		log.Logger().Debug("IntrospectAccessToken: failed to get token from store")
 		return IntrospectAccessToken200JSONResponse{}, err
```

Comment:
error is still logged, right?

Reply:
Nothing was logged, now there's some debug log. We may want to create an issue to 'fix' the logging for introspection calls?